### PR TITLE
[BUTTON] Radiobutton

### PR DESCRIPTION
**Pull request by epifaniovillaro**
## Issue Link
https://framgiaph.backlog.com/view/LMS-322

## Steps to reproduce
test: http://localhost:3000/demo/components
component location: shared/components/Radiobutton/index.tsx

## Affected Components / Functionalities / Page
demo/components/index.tsx

## Test Cases
_will update soon..._

## Notes
n/a

## Screenshots
image
image

### CODE REVIEW

**Review by ragg09-sunasterisk — COMMENTED** [review]
Please fix this, this is related to the actual task since I noticed that the radiobutton has the same behavior, I provided a clip of the error
image
radiobutton

**Review by ragg09-sunasterisk — APPROVED** [review]

**Comment by epifaniovillaro** [comment]
image

**Comment by ragg09-sunasterisk** [comment]
Comments: 2
Iterations: 0

File: client/src/shared/components/Radiobutton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+interface Props {
+  label?: string;
+  options: string[];
+  alignment?: string;
+  classname?: string;
+  onClickEvent: () => unknown;
+}
+
+const Radiobutton: React.FC<Props> = ({ label, options, alignment, classname, onClickEvent }: Props) => {
```
Comment (escbooster12-dev) [inline client/src/shared/components/Radiobutton/index.tsx:11]: - export the interface.
- specific interface name e.g. `CheckboxButtonProps`

File: client/src/shared/components/Radiobutton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, { Fragment } from 'react';
+
```
Comment (escbooster12-dev) [inline client/src/shared/components/Radiobutton/index.tsx:2]: - `RadioButton` not `Radiobutton`.
image

**Pull request merged**